### PR TITLE
Pack external stdlibs

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Purpose
$subject

## Approach
- Pack external stdlibs
- Parameterize stdlib and external dependency versions

## Test environment
SLP4

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -93,7 +214,22 @@ subprojects {
     }
     dependencies {
         /* Standard libraries */
+        ballerinaStdLibs "org.ballerinalang:http-ballerina:${stdlibHttpVersion}"
+        ballerinaStdLibs "org.ballerinalang:mime-ballerina:${stdlibMimeVersion}"
+        ballerinaStdLibs "org.ballerinalang:auth-ballerina:${stdlibAuthVersion}"
+        ballerinaStdLibs "org.ballerinalang:file-ballerina:${stdlibFileVersion}"
+        ballerinaStdLibs "org.ballerinalang:cache-ballerina:${stdlibCacheVersion}"
+        ballerinaStdLibs "org.ballerinalang:crypto-ballerina:${stdlibCryptoVersion}" //
```
Comment (chamil321) [inline build.gradle:222]:
```suggestion
        ballerinaStdLibs "org.ballerinalang:crypto-ballerina:${stdlibCryptoVersion}"
```

File: build.gradle [hunk 2]
```diff
@@ -93,7 +214,22 @@ subprojects {
...
         ballerinaStdLibs "org.ballerinalang:encoding-ballerina:${stdlibEncodingVersion}"
+        ballerinaStdLibs "org.ballerinalang:task-ballerina:1.1.0-SNAPSHOT"
```
Comment (chamil321) [inline build.gradle:224]: Shall we use the constant here for the version?

File: build.gradle (hunk 1 above)
Comment (chamil321) [reply build.gradle:222]: Shall we remove other `//` as well?

File: build.gradle [hunk 3]
```diff
@@ -36,7 +36,23 @@ ext.wso2SnakeYamlVersion = project.wso2SnakeYamlVersion
...
 ext.guavaVersion = project.guavaVersion
 ext.wso2CarbonMetricsVersion = project.wso2CarbonMetricsVersion
+ext.stdlibHttpVersion = project.stdlibHttpVersion
+ext.stdlibMimeVersion = project.stdlibMimeVersion
+ext.stdlibAuthVersion = project.stdlibAuthVersion
+ext.stdlibFileVersion = project.stdlibFileVersion
+ext.stdlibCacheVersion = project.stdlibCacheVersion
+ext.stdlibCryptoVersion = project.stdlibCryptoVersion
 ext.stdlibEncodingVersion = project.stdlibEncodingVersion
+ext.stdlibEncodingVersion = project.stdlibEncodingVersion
```
Comment (ldclakmal) [inline build.gradle:46]: Removing duplicate
```suggestion
ext.stdlibEncodingVersion = project.stdlibEncodingVersion
```

File: build.gradle (hunk 1 above)
Comment (BuddhiWathsala) [reply build.gradle]: Fixed in https://github.com/ballerina-platform/module-ballerina-websub/pull/17/commits/dffc9024569233a20a1e9b4b81cc35ed80ce5024

File: build.gradle (hunk 3 above)
Comment (BuddhiWathsala) [reply build.gradle]: Fixed in https://github.com/ballerina-platform/module-ballerina-websub/pull/17/commits/dffc9024569233a20a1e9b4b81cc35ed80ce5024

File: build.gradle (hunk 2 above)
Comment (BuddhiWathsala) [reply build.gradle]: Fixed in https://github.com/ballerina-platform/module-ballerina-websub/pull/17/commits/dffc9024569233a20a1e9b4b81cc35ed80ce5024

File: README.md [hunk 4]
```diff
@@ -1 +1,60 @@
-# ballerina-platform-module-ballerina-websub
\ No newline at end of file
+Ballerina WebSub Library
+===================
+
+  [![Build](https://github.com/ballerina-platform/module-ballerina-websub/workflows/Build%20master%20branch/badge.svg)](https://github.com/ballerina-platform/module-ballerina-websub/actions?query=workflow%3A%22Build+master+branch%22)
```
Comment (niveathika) [inline README.md:4]: Shall we rename this GitHub action as just build? Else the badge will be really long

https://github.com/ballerina-platform/module-ballerina-websub/blob/buddhi-dev/.github/workflows/build-master.yml#L1

**Comment by niveathika** [comment]
Please address the last comment, other than that LGTM

**Review by niveathika — APPROVED** [review]
Please address the last comment, other than that LGTM

File: README.md (hunk 4 above)
Comment (BuddhiWathsala) [reply README.md]: Fixed in here https://github.com/ballerina-platform/module-ballerina-websub/pull/17/commits/e9fc0613a847130c46dbb769159ab079eab38de2

**Pull request merged**